### ISSUE TITLE
DeS Collision with multiple lo materials not exporting properly (possibly only m07? Still an issue)

### ISSUE THREAD

**Issue by ArianeC7150** [Grimrukh/soulstruct-blender#89]
Collision files that use multiple lo materials fail at exporting. While they do export properly, one of the materials will not function within game. A good example of this is h0000b1 in m07_01.

**Comment by Grimrukh** [comment]
I'll investigate this shortly.

**Comment by Grimrukh** [comment]
Can you say more about "will not function"? Do you fall through that part of the collision? It has the wrong hitting sounds? Causes the game to crash? Etc.

**Comment by ArianeC7150** [comment]
Yeah sorry for not writing a more detailed explanation, you fall through the collision.

**Comment by Grimrukh** [comment]
I notice that the second lo material appears to be indexed 7, which I've never seen used before (note the lack of suffix like '<Wood>' after in Blender). My guess is that it's no longer a defined material in the final game - maybe try reassigning those faces to the other Lo material already present (`HKX 2 (Lo) <Stone>`)?

**Comment by ArianeC7150** [comment]
HKX 7 is the only snow material in the game as far as I'm aware, the issue is defining those faces to a different material would remove the proper snow footstep sounds. That solution wouldn't be the best for the first level especially, because almost the entire level is snow on the ground and walls made of stone.

The original hkx files within m07_01 work with both hkx 2 (Lo) <Stone> and hkx 7 without having this issue

**Comment by Grimrukh** [comment]
Gotcha, that's my bad, material 7 is totally snow and that's still the case in DS1. I've added it to the HKX material enum (in next release). So the issue definitely isn't that.

I'll have to do some testing for this on my end. It could be related to the fact that the vanilla version of this mesh is an ancient Havok 5.1 mesh type; SFB upgrades it to the standard format automatically on export, though if that were an issue (for some reason), I wouldn't expect just one lo-res mesh to not work... Will continue investigating.

**Comment by ArianeC7150** [comment]
something to add to this since I've still been looking into it. I've also been having the same issue when exporting collision from m08_00. I still don't fully understand this issue, but hopefully it'll be resolved sometime soon
